package.json

```diff
@@ -107,7 +109,10 @@
     "publish:staging:patch": "npm run publish:patch -- --environment staging --build-script build:staging",
     "publish:mainnet:major": "npm run publish:major -- --environment mainnet --build-script build:mainnet",
     "publish:mainnet:minor": "npm run publish:minor -- --environment mainnet --build-script build:mainnet",
-    "publish:mainnet:patch": "npm run publish:patch -- --environment mainnet --build-script build:mainnet"
+    "publish:mainnet:patch": "npm run publish:patch -- --environment mainnet --build-script build:mainnet",
```

Comment:
Super tiny nitpick: could we re-order these publish scripts as `mainnet -> rinkeby -> staging -> ropsten` to be in similar order to the build and start scripts?